packages/common/src/search/_internal/hubSearchChannels.ts

```diff
@@ -72,14 +77,19 @@ export const processSearchParams = (
     }
   });
   // Acceptable fields to use as filters
-  const filterProps: Record<string, any> = {};
-  const allowedFilterProps: Array<keyof ISearchChannels> = ["access", "groups"];
+  const filterProps: Record<string, string[]> = {};
+  const allowedFilterProps: Array<keyof ISearchChannels> = [
+    "access",
+    "groups",
+    "name",
+  ];
   // Find predicates that match acceptable filter fields
   query.filters.forEach((filter) => {
     filter.predicates.forEach((predicate) => {
       Object.keys(predicate).forEach((key: any) => {
-        if (allowedFilterProps.includes(key)) {
-          filterProps[key] = predicate[key];
+        const _key = mapKey(key);
```

Comment:
Minor, we could slightly simplify using an `object` vs `function` to facilitate the key mappings. Untested, but something like:

```ts
const keyMappings = {
  sortField: 'sortBy',
  term: 'name'
};

// ...

const { [key]: _key = key } = keyMappings;
```